js/achievements_highscores.js

```diff
@@ -330,7 +260,6 @@ localStorage.setItem('fortknightHS', JSON.stringify(highscoreDict));
 renderHighscores();
 }
 
-let highscoreDict;
 try {
 highscoreDict = JSON.parse(localStorage.getItem('fortknightHS'));
```

Comment:
This code seems to be a duplicate of line 43. Did you move the code and then forget to delete the copy?